specs/interop/superchain-config.md

```diff
@@ -0,0 +1,125 @@
+# Superchain Configuration Interop
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Pausability](#pausability)
+- [Storage Layout](#storage-layout)
+- [Interface and properties](#interface-and-properties)
+  - [`addDependency`](#adddependency)
+  - [`authorizedPortals`](#authorizedportals)
+  - [`dependencySet`](#dependencyset)
+  - [`dependencySetSize`](#dependencysetsize)
+- [Events](#events)
+  - [`DependencyAdded`](#dependencyadded)
+- [Invariants](#invariants)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The `SuperchainConfigInterop` contract extends `SuperchainConfig` to manage the dependency set and portal authorization
+for interoperability features.
+
+The contract is updated with a new `CLUSTER_MANAGER` role that has the ability to add chains to the dependency set.
```

Comment:
should we mention that the idea is to use this role just for the first addDependecy?

Reply:
To follow up on this, we will discuss it with OP tomorrow on the Lockbox Review Kickoff call